src/main/starlark/builtins_bzl/common/cc/cc_toolchain.bzl

```diff
@@ -342,5 +344,15 @@ The label of the rule providing <code>cc_toolchain_config_info</code>.""",
             default = semantics.BUILD_INFO_TRANLATOR_LABEL,
             providers = [OutputGroupInfo],
         ),
+        "_agg_ddi": attr.label(
```

Comment:
I'd suggest using a slightly longer name, so that it can be easier to understand it on a first read.

My preference: "aggregate_ddi", "generate_modmap"